src/amo/sagas/reviews.js

```diff
@@ -235,9 +253,25 @@ function* manageAddonReview(
     );
 
     yield put(setReview(reviewFromResponse));
+
+    if (savingRating) {
+      yield put(flashReviewMessage(SAVED_RATING));
+    }
+    if (savingReview) {
+      yield put(flashReviewMessage(SAVED_REVIEW));
+    }
+
+    // Make the message disappear after some time.
+    yield new Promise((resolve) =>
+      setTimeout(resolve, FLASH_SAVED_MESSAGE_DURATION),
+    );
+    yield put(flashReviewMessage(undefined));
```

Comment:
That's a bit weird to me, I'd rather write another action creator (`hideFlashReviewMessage`), even if it does the same thing under the hood.